src/components/pages/common/PageTitle.js

```diff
@@ -1 +1,31 @@
-// TODO: Write a reusable PageTitle component here using the Contact page's title.
+import React from 'react';
```

Comment:
You will want to write a corresponding unit test for this new component that you have just written. Since this is just a simple visual component with no crazy functionalities, you can just check if it is rendering correctly with snapshot testing.

Look at `SidebarHeader.test.js` in `src/components/sidebar` for an example of what you will have to do.